File: testdata/java/src/main/java/com/example/calc/StringUtils.java
package com.example.calc;

public final class StringUtils {

    private StringUtils() {
    }

    public static boolean isBlank(String s) {
        return s == null || s.trim().isEmpty();
    }

    public static String repeat(String s, int times) {
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < times; i++) {
            sb.append(s);
        }
        return sb.toString();
    }
}

File: testdata/csharp/Calc/StringUtils.cs
namespace Calc;

public static class StringUtils
{
    public static bool IsBlank(string s)
    {
        if (s == null)
        {
            return true;
        }
        return s.Trim().Length == 0;
    }

    public static string Repeat(string s, int times)
    {
        var result = "";
        for (var i = 0; i < times; i++)
        {
            result += s;
        }
        return result;
    }
}
